src/components/actions/FormSubmissionModal.js

```diff
@@ -37,38 +38,23 @@ const twitterUrl = ({ social }) => `https://twitter.com/${social.twitter}`;
 const NewsletterAndTwitterInner = ({ siteMetadata, classes }) => (
   <p>
     Learn more about Backstage via{' '}
-    <a
-      href={siteMetadata.newsletterUrl}
-      target="_blank"
-      rel="noopener noreferrer"
-      className={classes.link}
-    >
+    <Link to="/backstage-weekly/" className={classes.link}>
       our newsletter
-    </a>{' '}
```

Comment:
should it be nested like this? should the Link be totally nested within the anchor tag?

Reply:
I'm not sure I understand the comment. Could you explain a little more please?